### PR TITLE
Update winit and glutin

### PR DESCRIPTION
**Pull request by azriel91**
Just leaving this here while we wait for https://github.com/gfx-rs/gfx/pull/2468 to be merged.

It's merged!

### CODE REVIEW

**Comment by azriel91** [comment]
bors retry

**Comment by azriel91** [comment]
bors try

File: amethyst_input/src/button.rs [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
-    local_virtual_key_code::LocalVirtualKeyCode, scroll_direction::ScrollDirection,
-};
+use super::{controller::ControllerButton, scroll_direction::ScrollDirection};
 
 /// A Button is any kind of digital input that the engine supports.
 #[derive(Eq, PartialEq, Debug, Copy, Clone, Serialize, Deserialize)]
 pub enum Button {
     /// Virtual Keyboard keys, use this when the letter on the key matters
     /// more than the position of the key.
-    Key(#[serde(with = "LocalVirtualKeyCode")] VirtualKeyCode),
```
Comment (torkleyy) [inline amethyst_input/src/button.rs]: Thanks a bunch to the one who made this possible!

**Comment by torkleyy** [comment]
bors try

**Comment by torkleyy** [comment]
Not funny, bors.

bors r=TimonPost,LucioFranco,torkleyy

**Comment by torkleyy** [comment]
Does this fix any of the current windowing bugs? I think there was something blocked by this..

**Comment by LucioFranco** [comment]
@torkleyy I don't think it fixes the 10.14 bug that we had to work around but I can verify this later.

**Pull request merged**